docs/usage/configuration/config_documentation.md

```diff
@@ -1380,6 +1380,22 @@ rc_joins:
     burst_count: 12
 ```
 ---
+### `rc_joins_per_room`
```

Comment:
Would you mind adding info here about what release this config option will be added in?